src/release_artifacts_resources/ios/cdk/cdk/credential_rotation/utils/lambda_constants.py

```diff
@@ -1,15 +1,27 @@
-# Used as lambda environment variable for the key that holds
-# CircleCI api key
+# Lambda environment variable for the key that holds
+# SecretsManager CircleCI api key
```

Comment:
Can you clarify this? Is it:

```python
# Lambda environment variable key, whose value is the
# SecretsManager secret name of the CIrcleCI API key
```

or something else?

Reply:
Yes, updated